docsrc/imap/reference/manpages/configs/cyrus.conf.rst

```diff
@@ -55,6 +55,13 @@ processes listed in **START** are expected to end and clean up after
 themselves, while those in **DAEMON** will be shutdown by
```

Comment:
I think I'd like to see this part say something like,

> Similarly, processes listed in START will be run to completion before any SERVICES are started, while those in DAEMON will be managed by master.

(n.b. one of the things DAEMON does that START does not is restart the process if it exits early or crashes, just like a SERVICE minus the socket.  It's not just about shutdowns.)

Reply:
Similarly, down under the START heading (github won't let me add a comment there because it's too far away from the actual diff), maybe get rid of the "or backgrounded themselves"?  And then add some sort of "historical note" that prior to v3, non-service daemons like idled were started from START but would background themselves, but now they are better managed through the DAEMON section?

Reply:
@elliefm This has been fixed in d7ff32f

Thanks!
-nic